Add `GitHub` updates on the release note (#8915)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

- Introduced support for connecting to Candid's Charity Check, Essentials and Premier REST APIs.

#### `github` package 

> - Introduced support for connecting to GitHub REST API version 2022-11-28, transitioning from GraphQL previously used.

Reviewer: Shall we mention something like this ?
```suggestion
- Introduced support for connecting to GitHub REST API version 2022-11-28, replacing the GitHub GraphQL API based connector.
```
